lib/Cmts.ml

```diff
@@ -73,7 +73,7 @@ end = struct
                    [ { pstr_desc=
                          Pstr_eval
                            ( { pexp_desc=
-                                 Pexp_constant (Pconst_string (doc, None))
+                                 Pexp_constant (Pconst_string (doc, Location.none, None))
```

Comment:
this is already the `pexp_loc` below and the `attr_loc` above so I think this is the right solution